### PR TITLE
Fix/new line

### PR DESCRIPTION
**Pull request by bryan-robitaille**
# Summary | Résumé

This PR introduces a fixed special character identifier that will allow line breaks in long descriptions and rich text.
`<br>` in the `descriptionEn` or `descriptionFr` will now correctly display line breaks.

| Before                                          | After                                        |
| ----------------------------------------------- | -------------------------------------------- |
|Screen Shot 2021-11-26 at 8 00 59 AM| Screen Shot 2021-11-26 at 8 02 07 AM|

# Test instructions | Instructions pour tester la modification

Create a form or edit an existing text form with a long description.  Previously the `/n/r` or other CRLF symbols were not properly rendering new line breaks in both the richText and the Description components.  Add a '<br>` in the description where you want the line break to occur.

For component descriptions (not richText) the parent div wraps the paragraph elements with the associated ID for the input's araiDescribedBy thus allowing for a11y compliance.

# Pull Request Checklist

Please complete the following items in the checklist before you request a review:

- [x] Have you completely tested the functionality of change introduced in this PR? Is the PR solving the problem it's meant to solve within the scope of the related issue?
- [x] The PR does not introduce any new issues such as failed tests, console warnings or new bugs.
- [x] If this PR adds a package have you ensured its licensed correctly and does not add additional security issues?
- [x] Is the code clean, readable and maintainable? Is it easy to understand and comprehend.
- [x] Have you removed unnecessary lines such as `console.log` statements?
- [x] Does your code have adequate comprehensible comments? Do new functions have [docstrings](https://tsdoc.org/)?
- [x] Have you modified the change log and updated any relevant documentation?
- [x] Is there adequate test coverage? Both unit tests and end-to-end tests where applicable?
- [x] If your PR is touching any UI is it accessible? Have you tested it with a screen reader? Have you tested it with automated testing tools such as axe?

### CODE REVIEW

File: components/forms/Description/Description.tsx [hunk 1]
```diff
@@ -11,11 +11,18 @@ export const Description = (props: DescriptionProps): React.ReactElement => {
...
-
   return (
-    <p data-testid="description" className={classes} id={`desc-${id}`}>
-      {children}
-    </p>
+    <div id={`desc-${id}`}>
+      {children
+        ? children.split("<br>").map((child, index) => {
+            return (
+              <p data-testid="description" className={classes} key={index}>
```
Comment (jeberhardt) [inline components/forms/Description/Description.tsx:19]: Strange to replace `<br>` with `<p></p>`, as they are semantically differently tags. Is it possible to split on `<br>` within the string, as you are currently doing, and replace it with an actual `<br>` tag?

**Review by Moro-Code — DISMISSED** [review]
Looks good. But need to add to mdx files for documentation otherwise this is going to get lost.

File: components/forms/Description/Description.tsx (hunk 1 above)
Comment (bryan-robitaille) [reply components/forms/Description/Description.tsx:19]: Good idea.  Implemented

**Comment by bryan-robitaille** [comment]
Added documentation to MDX file

**Review by jeberhardt — APPROVED** [review]

**Pull request merged**